### PR TITLE
rbd-mirror: admin socket commands to start/stop/restart mirroring

### PR DESCRIPTION
**Pull request by trociny**

### CODE REVIEW

File: src/tools/rbd_mirror/ImageReplayer.cc [hunk 1]
```diff
@@ -181,6 +181,14 @@ ImageReplayer<I>::ImageReplayer(Threads *threads, RadosRef local, RadosRef remot
   m_replay_handler(nullptr),
   m_asok_hook(nullptr)
 {
+  // Register asok commands using a temporary "remote_pool_id/remote_image_id"
+  // name.  When the pool and image names become known on start the asok
+  // commands will be re-registered using "remote_pool_name/remote_image_name"
+  // name.
+
+  CephContext *cct = static_cast<CephContext *>(m_local->cct());
+  m_name = stringify(remote_pool_id) + "/" + remote_image_id;
```
Comment (dillaman) [inline src/tools/rbd_mirror/ImageReplayer.cc]: Can we use the global image ID instead?

Comment (dillaman) [inline src/tools/rbd_mirror/ImageReplayer.cc]: Pool name should be known / consistent

**Comment by trociny** [comment]
@dillaman rebased after changing the temporary name to pool_name/global_image_id.

**Comment by dillaman** [comment]
@trociny I ran this locally and I see the following unittest_rbd_mirror failures:

> [  FAILED  ] TestImageReplayer.ErrorJournalReset
> [  FAILED  ] TestImageReplayer.ErrorNoJournal
> [  FAILED  ] TestImageReplayer.StartStop
> [  FAILED  ] TestImageReplayer.WriteAndStartReplay
> [  FAILED  ] TestImageReplayer.StartReplayAndWrite
> [  FAILED  ] TestImageReplayer.NextTag
>

**Comment by trociny** [comment]
@dillaman sorry, fixed. Thanks

**Pull request merged**